pandas/core/arrays/categorical.py

```diff
@@ -2025,8 +2025,8 @@ def __unicode__(self):
         return result
 
     def __repr__(self):
-        # We want PandasObject.__repr__, which dispatches to __unicode__
-        return super(ExtensionArray, self).__repr__()
+        # We want to bypass ExtensionArray.__repr__.
+        return str(self)
```

Comment:
This ends up in both master abd this PR to call ``self.__unicode__``. ``__unicode__`` isn't needed anymore so should be cleaned up in a future PR.